lib/bgs/service.rb

```diff
@@ -16,6 +16,10 @@ class Service
     # (I = Input, U = Update, D = Delete)
     JOURNAL_STATUS_TYPE_CODE = 'U'
 
+    # It appears that a find_ch33_dd_eft that returns empty bank account information
+    # will set the routing number field to '0' instead of 'nil', at least in certain cases
+    EMPTY_ROUTING_NUMBER = '0'
```

Comment:
Would we want to freeze this string?
```suggestion
    EMPTY_ROUTING_NUMBER = '0'.freeze
```

Reply:
Ahh, right, must be the `frozen_string_literal: true` kicking in. Sorry about that.